Foundation/NSURLSession/http/HTTPURLProtocol.swift

```diff
@@ -495,6 +498,16 @@ extension _HTTPURLProtocol: _EasyHandleDelegate {
         }
     }
 
+    fileprivate func notifyDelegate(aboutSentData count: Int64) {
```

Comment:
@ianpartridge I agree that the name should be better. However, it may not make sense to have Data as a parameter since the delegate call only needs the length of the data that was uploaded. Also, the caller of this method has the data in the form of`DispatchData`. I reckon there'll be some unnecessary cost of converting DispatchData to Data.